### PR TITLE
LPS-146001 [CP2.0] Add CP2EnterpriseSearch tests

### PR DESCRIPTION
**Pull request by luanakarla**

### CODE REVIEW

**Comment by luanakarla** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2EnterpriseSearch.testcase [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	@priority = "5"
+	test CanViewListOfStartAndEndDatesActiveSubscriptionTerms {
+
+		// TODO LPS-146022 CanViewListOfStartAndEndDatesActiveSubscriptionTerms pending implementation
+
+	}
+
+	@description = "Automation ID: LPS-146021 | Test Summary: Verify that by clicking on subscriptions the user can select from a drop-down list his LES subscriptions"
+	@priority = "5"
+	test SeeListOfLessSubscriptions {
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2EnterpriseSearch.testcase:56]: Hi @luanakarla is it Less or LESSubscriptions?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2EnterpriseSearch.testcase [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+	@description = "Automation ID: LPS-146022 | Test Summary: Verify that once a subscription was selected, the next drop-down is populated a list of start and end dates"
+	@priority = "5"
+	test CanViewListOfStartAndEndDatesActiveSubscriptionTerms {
+
+		// TODO LPS-146022 CanViewListOfStartAndEndDatesActiveSubscriptionTerms pending implementation
+
+	}
+
+	@description = "Automation ID: LPS-146021 | Test Summary: Verify that by clicking on subscriptions the user can select from a drop-down list his LES subscriptions"
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2EnterpriseSearch.testcase:54]: Please change "his" to "their". 

"Their" is an impersonal pronoun. It is more adequate than "his"

Comment (luanakarla) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2EnterpriseSearch.testcase]: Done.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2EnterpriseSearch.testcase (hunk 1 above)
Comment (luanakarla) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2EnterpriseSearch.testcase]: Done.

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:forward

**Pull request closed without merging**